cmd/server/assets/admin/info.html

```diff
@@ -19,8 +19,23 @@
           <dt>Build ID</dt>
           <dd>{{.buildID}}</dd>
 
-          <dt>Build Tag</dt>
+          <dt>Build tag</dt>
           <dd>{{.buildTag}}</dd>
+
+          <dt>Token signing key</dt>
```

Comment:
can we display the allowed keys as well - there should be a 24h + overlap where more than one key is valid?

Reply:
Done